README.md

```diff
@@ -1,101 +1,480 @@
+
+**Oracle**: The Oracle connector requires Oracle client libraries 11.2+ and can connect to Oracle Database Server 9.2+.
 
 ## Installation
 
-To simplify the installation of [node-oracledb](https://github.com/oracle/node-oracledb) module and Oracle instant clients,
-we introduce [loopback-oracle-installer](https://github.com/strongloop/loopback-oracle-installer) as a dependency which installs
-and configures node-oracledb (oracledb) upon `npm install`.
+In your application root directory, enter this command to install the connector:
+
+```shell
+$ npm install loopback-connector-oracle --save
+```
+
+If you create a Oracle data source using the data source generator as described below, you don’t have to do this, since the generator will run npm install for you.
+
+See [Installing the Oracle connector](http://loopback.io/doc/en/lb3/Installing-the-Oracle-connector.html) for further installation instructions.
+
+To simplify the installation of [node-oracledb](https://github.com/oracle/node-oracledb) module and Oracle instant clients, use [loopback-oracle-installer](https://github.com/strongloop/loopback-oracle-installer) as a dependency to install and configure `node-oracledb` (oracledb) upon `npm install`.
 
 Please note `config.oracleUrl` is the property to define the base URL to download the corresponding node-oracle (oracledb) bundle for the local
 environment.
 
 The bundle file name is `loopback-oracle-<platform>-<arch>-<version>.tar.gz`. The `version` is the same as the `version` in package.json.
 
-    "dependencies": {
-        "loopback-oracle-installer": "git+ssh://git@github.com:strongloop/loopback-oracle-installer.git",
-             ...
-    },
-    "config": {
-        "oracleUrl": "http://7e9918db41dd01dbf98e-ec15952f71452bc0809d79c86f5751b6.r22.cf1.rackcdn.com"
-    },
+```javascript
+  "dependencies": {
+      "loopback-oracle-installer": "git+ssh://git@github.com:strongloop/loopback-oracle-installer.git",
+           ...
+  },
+  "config": {
+      "oracleUrl": "http://7e9918db41dd01dbf98e-ec15952f71452bc0809d79c86f5751b6.r22.cf1.rackcdn.com"
+  },
+  ...
+```
 
-**The `oracleUrl` can be overridden via LOOPBACK_ORACLE_URL environment variable.**
+You can override the `oracleUrl` setting with the LOOPBACK_ORACLE_URL environment variable.
 
 For example, the full URL for v1.5.0 for MacOSX is:
 
 http://7e9918db41dd01dbf98e-ec15952f71452bc0809d79c86f5751b6.r22.cf1.rackcdn.com/loopback-oracle-MacOSX-x64-1.5.0.tar.gz
 
-`libaio` library is required on Linux systems:
+The `libaio` library is required on Linux systems:
 
-* On Ubuntu/Debian
+On Ubuntu/Debian, get it with this command:
 
-        ﻿sudo apt-get install libaio1
+```
+sudo apt-get install libaio1
+```
 
-* On Fedora/CentOS/RHEL
+On Fedora/CentOS/RHEL, get it with this command:
 
-        ﻿sudo yum install libaio
+```
+sudo yum install libaio
+```
 
+**IMPORTANT**: Ensure `c:\instantclient_12_1\vc10` comes before `c:\instantclient_12_1`.
```

Comment:
OK, I removed that sentence. Thanks again @cjbj.
Unless there are more review comments, I'll merge this tomorrow.